### ISSUE TITLE
Electrons not being announced by VoiceOver

### ISSUE THREAD

**Issue by jobara** [phetsims/john-travoltage#210]
I'm able to navigate to the leg and rub the foot across the carpet using VoiceOver. Visually I can see that electrons are accumulating, but there is not announcement of this by VoiceOver. 

Using Safari 10.0.3 on macOS 10.12.3 with VoiceOver

**Comment by terracoda** [comment]
@jobara, I tested on the same platforms, I hear the electron count sometimes, but not all the time. It seems to go back and forth sometime lots of leg position descriptions then lots of electron counts.

**Comment by jobara** [comment]
@terracoda for the most part i was moving the leg slowly. 1 key press, wait for the announcement, then press again. I did hear the position announced twice. Were you interacting the same way? I wonder if it maybe related to how quickly the leg is moved, interruptions with announcements, or something else? @jessegreenberg could it be related to the event emitter for the electrons?

**Comment by jessegreenberg** [comment]
>  I wonder if it maybe related to how quickly the leg is moved, interruptions with announcements,

That seems likely, but I haven't been able to reproduce this myself.  I am using Safari 9.0.1 with OSX 10.9.5.

**Comment by jessegreenberg** [comment]
>  could it be related to the event emitter for the electrons?

Possibly, though I thought we only added emitters for when charges exit the body.  Ill see if I can grab a newer version of Safari and macOS

**Comment by jessegreenberg** [comment]
@phet-steele has latest macOS and was able to reproduce this bug.

**Comment by jessegreenberg** [comment]
Since it isn't happening on older VoiceOver/Safari, this seems more like a VoiceOver issue with interrupting announcements.

**Comment by terracoda** [comment]
@jobara, @jessegreenberg I was going fast then slow. The experience is so much slower when using VoiceOver than when using the keyboard alone. With keyboard alone, the slider works perfectly. 

I thought I would try using the mouse while also using VoiceOver. At first I had a long run where I only got foot position. When I clicked away from the foot, I got the electron count. After a discharge and a reset, I only got electron information. ....Oh, I see, I have to tab to the sliders and put keyboard focus on the leg or the hand to hear position information when using the mouse. At first, I always heard discharges, but after using the arm I am no longer hearing discharge information. Hmm, seems that descriptions are dropped from the VO cue when using the mouse rather erratically.

I'm actually now having trouble operating the sliders at all when using VoiceOver. Maybe I have a weird keyboard setting turned on.

**Comment by jobara** [comment]
@terracoda I find at times that things just stop working the way I expect in VoiceOver and I just reset all of it's settings, and things start to work again. I'm not sure why this is. If it's something I'm doing that's causing it, or some bug in VoiceOver.

**Comment by terracoda** [comment]
@jobara, I find the same thing. BTW, are you using VO keys and Arrow keys to operate the sliders or just the Arrow keys alone? My keyboard focus switches when I use the Arrow keys, though in my previous test I was able to operate the slider.

**Comment by jessegreenberg** [comment]
@mbarlow12 and I found that the alerts can be heard on VoiceOver with macOS when the aria-atomic attribute is removed.  I don't actually remember why that attribute is there, @jobara can it be removed?

**Comment by jobara** [comment]
@jessegreenberg here's the write up on aria-atomic ( https://www.w3.org/TR/wai-aria/states_and_properties#aria-atomic ). I'm guessing it's for cases where only a single value in a structure is changed, example if the alert is "electrons: 5" and only the 5 is changing. You'd want it to read "electrons: 6" and not just "6". In our case if we're changing the entire node anyways, it shouldn't make a difference and removing "aria-atomic" should be fine.

**Comment by jessegreenberg** [comment]
Excellent, thanks @jobara.

**Comment by jessegreenberg** [comment]
Updates are coming through again consistently, closing.
